lib/openstax/exchange/client/client.rb

```diff
@@ -0,0 +1,67 @@
+module OpenStax
+  module Exchange
+    module Client
+
+      @use_real_client = true
+
+      def self.configure
+        yield configuration
+      end
+
+      def self.configuration
+        @configuration ||= ClientConfiguration.new
+      end
+
+      def self.reset!
+        @client = nil
+      end
+
+      def self.use_real_client
+        @use_real_client = true
+      end
+
+      def self.use_fake_client
+        @use_real_client = false
+      end
+
+      def self.use_real_client?
+        !!@use_real_client
+      end
+
+      def self.create_identifier
+        begin
+          client.create_identifier
+        rescue StandardError => error
+          raise ClientError.new("create_identifier failure", error)
+        end
+      end
+
+      def self.create_multiple_choice(*args)
```

Comment:
I'm fine with either having the `_event` suffix or not, as long as it's consistent for future methods.